src/Core/Migration/Migration1572264837AddCacheId.php

```diff
@@ -16,8 +16,11 @@ public function getCreationTimestamp(): int
     public function update(Connection $connection): void
     {
         $connection->executeUpdate('DELETE FROM app_config');
-        $connection->exec('ALTER TABLE app_config ADD PRIMARY KEY (`key`)');
-
+        try {
+            $connection->exec('ALTER TABLE app_config ADD PRIMARY KEY (`key`)');
+        } catch (DBALException $e) {
```

Comment:
Hey @powolnymarcel, you were missing an import for Doctrine\DBAL\DBALException here
The migration broke, because the catch block was never reached.

Fixed it on my side already to push the pr forward, so just fyi

Reply:
Thanks :)